sixtyfps_runtime/corelib/item_rendering.rs

```diff
@@ -38,7 +38,7 @@ impl CachedRenderingData {
     pub fn get_or_update<T: Clone>(
         &self,
         cache: &RefCell<RenderingCache<T>>,
-        update_fn: impl FnOnce() -> T,
+        update_fn: impl FnMut() -> T,
```

Comment:
Why the change?  Isn't  FnOnce stronger than FnMut ?

Reply:
See comment in `lib.rs`, there this allows replacing a use of interior mutability. But maybe that's not worth it and it's better to keep this `FnOnce`?

Reply:
I think I spoke too soon. Not sure why, but if I change it back now it works :)